rafthttp/transport.go

```diff
@@ -78,13 +81,24 @@ type Transporter interface {
 	Stop()
 }
 
-type transport struct {
-	roundTripper http.RoundTripper
-	id           types.ID
-	clusterID    types.ID
-	raft         Raft
-	serverStats  *stats.ServerStats
-	leaderStats  *stats.LeaderStats
+// Transport is the implementation of Transporter. It provides the functionality
+// to send raft messages to peers, and receive raft messages from peers.
+// The Handler in Transport needs to serve requests sent to peer URLs of the
```

Comment:
I cannot really understand line 86 and line 87. I am not sure which Handler(There is not handler field in Transport struct) are you referring to and I am not sure what does "work well" really mean. Should I provide a Handler to Transport? Or is it an implementation detail that I should not care about?